### DISCUSSION TITLE
BIDS

### DISCUSSION THREAD

**Discussion by giscus** [bids-standard/bids-website#406] [General]
# BIDS

https://bids-website.readthedocs.io/en/latest/index.html

**Comment by Remi-Gau** [comment]
The first image of the landing page is not visible when in light mode.

**Reply by bendhouseart** [reply]
Do we want to disable light mode until it's fixed? I know it's nice to have the option, but I got pretty frustrated flicking back and forth with it a week or two ago while pushing changes to `main`.

**Reply by Remi-Gau** [reply]
while we got the image fixed I would say yes, let's disable the light mode

also light attracts bugs so defo something we should do

**Reply by ericearl** [reply]
@Remi-Gau fixed it! My hero. Should we close this discussion then? And how would we do that as a process/procedure regularly when discussions become resolved?

**Reply by Remi-Gau** [reply]
The outcome is still meh in dark mode, so I would open an issue from this discussion thread so we can better track it.

**Comment by VisLab** [comment]
Overall:  An order of magnitude improvement in organizing information about the BIDS ecosystem. Thanks for the hard work by the BIDS maintainers! The search is great.

Here are some minor comments -- none of which I am wedded too -- just my general impressions:

1.  Might be better to have Getting Started come after About on the top menu bar.
2.  Since the About is the landing page a lead paragraph about exactly what BIDS is would be useful. It should emphasize some key points:   a) BIDS is a standardized method of organizing data
                    b) a large community of researchers contributed to make sure that it includes what is needed to make the data useable
                    c) A large ecosystem of tools and datasets based on BIDS has emerged, making BIDS the foundation for much neuroimaging analysis.
                     d) BIDS has a large, supportive user community to help new users get started.

3)  Maybe the file system picture could come under the title What does BIDS look like?   The information about openFMRI maybe should go under a History section.

4)  I think new users are confused about what the BIDS specification is versus what the validators and other stuff.

As a general item on style, The titles on the left sidebar were a little confusing to navigate.  My minor suggestion is that the type face of the top sidebar title (which corresponds to an item on the top menu bar) should have a larger type-face and remain in a fixed color. 

I have some specific requests on certain pages.  Should I post issues?

**Reply by Remi-Gau** [reply]
yes if you have very specific points, I think it is best to open issues indeed.

thanks already for taking the time to have a look and let us know.
